validator/src/commands/staked_nodes_overrides/mod.rs

```diff
@@ -1,11 +1,26 @@
 use {
-    crate::{admin_rpc_service, cli::DefaultArgs},
+    crate::{admin_rpc_service, cli::DefaultArgs, commands::FromClapArgMatches},
     clap::{App, Arg, ArgMatches, SubCommand},
     std::{path::Path, process::exit},
 };
 
+const COMMAND: &str = "staked-nodes-overrides";
+
+#[derive(Debug, PartialEq)]
+pub struct StakedNodesOverridesArgs {
+    pub path: String,
```

Comment:
This should seemingly be a `PathBuf`; I can see the argument for changing the type separate from this PR. Any interest in adjusting the type to be a `PathBuf` in another PR, and then rebasing this PR on top of that one when it lands ?

Reply:
I think this is a newly defined struct. maybe we can just call it a PathBuf now! [40a3875](https://github.com/anza-xyz/agave/pull/4903/commits/40a387545e8990c8ed4e02c275cd2af7c5b76783)

Reply:
Ahh shoot, I see `set_staked_nodes_overrides` wants a String. Sorry to reverse on you here, but let's keep it as a String instead of doing String ==> PathBuf ==> String conversion